lib/compiler/src/compile.erl

```diff
@@ -476,19 +514,27 @@ Available options:
   `Module:parse_transform/2` to be applied to the parsed code before the code is
   checked for errors.
 
-- **`from_abstr`** - The input file is expected to contain Erlang terms
-  representing forms in abstract format (default file suffix ".abstr"). Note
-  that the format of such terms can change between releases.
+- **`to_abstr`** - Dumps the terms representing the [abstract
+  format](`e:erts:absform.html`), after all source code
```

Comment:
to_abstr dumps the terms after parse transformations and linting and the remove_doc pass. The undocumented option to_exp dumps the terms after all source level transformations are done, such as expanding records to tuples.

The from_abstr option currently picks up after the docs are stripped, so you won't be able to compile from abstr with docs right now - this should probably be changed so that the docs passes are done within the abstr_passes() section in compile.erl.

Reply:
I've corrected the documentation for `to_abstr` and added documentation for `to_exp`.

> The from_abstr option currently picks up after the docs are stripped, so you won't be able to compile from abstr with docs right now - this should probably be changed so that the docs passes are done within the abstr_passes() section in compile.erl.

I will try to do that in a separate pull request.